dependabot-auto-merge/babel.config.js

```diff
@@ -4,6 +4,15 @@ const config = {
 		[ '@babel/preset-env', { targets: { node: 'current' } } ],
 		'@babel/preset-typescript',
 	],
+	env: {
+		test: {
+			plugins: [
+				// see https://github.com/facebook/jest/issues/936#issuecomment-821944391
+				// Jest has no good way of partially mocking a module
+				'explicit-exports-references',
```

Comment:
`Jest` can't do partial mocks properly. The one documented [here](https://jestjs.io/docs/mock-functions#mocking-partials) will have the original functions call other original functions rather than the mocked function due to imports still pointing to the old references.

This plugin helps fix this. I'm unsure of the recommendation here, but I prefer using this module rather than adding functions into an object (the test bridge object), and then calling it through the object. It feels pretty weird to export an object just so that we can test. Also, what if another developer forgot about this, accidentally call the function directly rather than the test bridge, and have to spend hours to realize this `Jest` gotcha?